src/js/node/child_process.ts

```diff
@@ -2,6 +2,7 @@
 const EventEmitter = require("node:events");
 const StreamModule = require("node:stream");
 const OsModule = require("node:os");
+const FsModule = require("node:fs");
```

Comment:
extremely expesnive, lets avoid loading this when not 100% necessary

we should honestly do the same for node:os